docs/install.md

```diff
@@ -270,6 +270,8 @@ $ kubectl create -f https://github.com/intel/pmem-csi/raw/devel/deploy/operator/
 ```
 The operator gets deployed in a namespace called 'pmem-csi' which gets created by that YAML file.
 
+**WARNING:** Using this YAML to stop(`kubectl delete -f`) the running operator is not recommended as it deletes the `pmem-csi` namespace object that results in bringing down the PMEM-CSI driver pods if any that were deployed by the operator.
```

Comment:
Additional explanations in round brackets look odd. How about this?

```
**WARNING:** This YAML file cannot be used to stop just the operator while 
keeping the PMEM-CSI deployments running. That's because 
something like `kubectl delete -f pmem-csi-operator.yaml` will delete the
`pmem-csi` namespace which then also causes all PMEM-CSI deployments that
might have been created in that namespace to be deleted.
```

Reply:
Agree, and changed as suggested. Thanks.